CrazyCanvas/Include/ECS/Systems/Player/WeaponSystem.h

```diff
@@ -106,6 +106,6 @@ class WeaponSystem : public LambdaEngine::System
 	static LambdaEngine::TUniquePtr<WeaponSystem> s_Instance;
 
 private:
-	float	m_ZeroDist	= 5.0f;
-	float	m_YAngle	= 0.0f;
+	float	m_ZeroDist	= 8.0f;
+	float	m_YAngle	= 5.f;
```

Comment:
defining convention, either 5.0f or 8.f ^^

Reply:
you are right! fixed